asvdb/asvdb.py

```diff
@@ -420,15 +422,17 @@ def __readResults(self, infoOnly=False, filterByInfoObjs=None):
                                       f"file: {resultsFile.as_posix()} "
                                       f"invalid name\"{benchmarkName}\", skipping.")
                                 continue
-
-                            benchmarkSpec = bDict[benchmarkName]
+                            benchmarkSpec = bDict.get(benchmarkName)
```

Comment:
I'm wondering if it's better if this crashes with a `KeyError` here like it did before, since this could now return `None` and continue on until something tries to use it.